apps/modernization-ui/src/apps/patient/add/extended/nav/AddPatientExtendedNav.tsx

```diff
@@ -13,7 +13,7 @@ export const AddPatientExtendedNav = () => {
                     <a href="#identification">Identification</a>
                     <a href="#section-Race">Race</a>
                     <a href="#ethnicity">Ethnicity</a>
-                    <a href="#section-SexAndBirth">Sex & birth</a>
+                    <a href="#sexAndBirth">Sex & birth</a>
```

Comment:
question: `sex-and-birth` or `sexAndBirth`?  For some reason kebab case looks better in urls

Reply:
agreed. Will see if Henry will update those as part of his nav component PR